packages/core/src/sync-store/index.ts

```diff
@@ -1273,212 +1048,19 @@ export const createSyncStore = ({
     db.wrap({ method: "pruneByChain" }, () =>
       db.transaction().execute(async (tx) => {
         await tx
-          .with("deleteLogFilter(logFilterId)", (qb) =>
-            qb
-              .selectFrom("logFilterIntervals")
-              .innerJoin("logFilters", "logFilterId", "logFilters.id")
-              .select("logFilterId")
-              .where("chainId", "=", chainId)
-              .where("startBlock", ">=", fromBlock.toString()),
-          )
-          .deleteFrom("logFilterIntervals")
-          .where(
-            "logFilterId",
-            "in",
-            ksql`(SELECT "logFilterId" FROM ${ksql.table("deleteLogFilter")})`,
-          )
+          .deleteFrom("interval")
+          .where("chain_id", "=", chainId)
+          .where("start_block", ">=", fromBlock.toString())
```

Comment:
This query would be slightly more complex with multirange types

Reply:
I ended up just changing the behavior of `disableCache` so that it doesn't ever read or write the `interval` table